packages/expo-module-template/src/index.ts

```diff
@@ -0,0 +1,9 @@
+import { NativeModulesProxy } from 'expo-core';
+
+const { ExpoModuleTemplate } = NativeModulesProxy;
+
+export { default as ExpoModuleTemplateView } from './ExpoModuleTemplateView';
+
+export async function someGreatMethod() {
```

Comment:
```suggestion
export async function someGreatMethodAsync(options: any) {
```

or a type matching the docs